### PR TITLE
Not so auto update

### PR DESCRIPTION
**Pull request by visr**
The monthly updates failed. This does them manually and fixes the reasons they failed.

### CODE REVIEW

**Comment by visr** [comment]
@evetion do we need to use another module?
```
15:25:43   ERROR: Unable to locate a modulefile for 'gcc/11.3.0'
```

And have you seen such a PicklingError before?

```
15:25:45   ✨ Pixi task (generate-testmodels in dev): python utils/generate-testmodels.py
15:25:54   Traceback (most recent call last):
15:25:54     File "/opt/teamcityagent/work/ecd2b8f9b25b1609/ribasim/utils/generate-testmodels.py", line 44, in <module>
15:25:54       for model_name in p.imap_unordered(generate_model_partial, models):
15:25:54                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
15:25:54     File "/opt/teamcityagent/work/ecd2b8f9b25b1609/ribasim/.pixi/envs/dev/lib/python3.12/multiprocessing/pool.py", line 873, in next
15:25:54       raise value
15:25:54   multiprocessing.pool.MaybeEncodingError: Error sending result: '<multiprocessing.pool.ExceptionWithTraceback object at 0x7fc8414cf3e0>'. Reason: 'PicklingError("Can't pickle <class 'ribasim.input_base.TableModel[BasinProfileSchema]'>: attribute lookup TableModel[BasinProfileSchema] on ribasim.input_base failed")'
15:25:54   Process exited with code 1
```

**Comment by visr** [comment]
I updated the module load command to the currently available module, and updated to GCC 12. Also fixed some mypy unused ignores. But there seems to be several more issues that need to be fixed.

This looks like an installation failure of typeguard that I see locally. Explicitly adding with pixi didn't seem to help:

```
pixi run generate-testmodels

  File "A:\repo\ribasim\Ribasim\python\ribasim\ribasim\geometry\area.py", line 1, in <module>
    import pandera as pa
  File "A:\repo\ribasim\Ribasim\.pixi\envs\dev\Lib\site-packages\pandera\__init__.py", line 10, in <module>
    import pandera.backends
  File "A:\repo\ribasim\Ribasim\.pixi\envs\dev\Lib\site-packages\pandera\backends\__init__.py", line 4, in <module>
    import pandera.backends.base.builtin_checks
  File "A:\repo\ribasim\Ribasim\.pixi\envs\dev\Lib\site-packages\pandera\backends\base\builtin_checks.py", line 14, in <module>
    from pandera.api.checks import Check
  File "A:\repo\ribasim\Ribasim\.pixi\envs\dev\Lib\site-packages\pandera\api\checks.py", line 17, in <module>
    from pandera.strategies.base_strategies import SearchStrategy
  File "A:\repo\ribasim\Ribasim\.pixi\envs\dev\Lib\site-packages\pandera\strategies\__init__.py", line 3, in <module>
    from pandera.strategies.pandas_strategies import *
  File "A:\repo\ribasim\Ribasim\.pixi\envs\dev\Lib\site-packages\pandera\strategies\pandas_strategies.py", line 42, in <module>
    from pandera.engines import numpy_engine, pandas_engine
  File "A:\repo\ribasim\Ribasim\.pixi\envs\dev\Lib\site-packages\pandera\engines\pandas_engine.py", line 33, in <module>
    import typeguard
  File "A:\repo\ribasim\Ribasim\.pixi\envs\dev\Lib\site-packages\typeguard\__init__.py", line 21, in <module>
    from ._importhook import ImportHookManager as ImportHookManager
  File "A:\repo\ribasim\Ribasim\.pixi\envs\dev\Lib\site-packages\typeguard\_importhook.py", line 35, in <module>
    OPTIMIZATION = "typeguard" + "".join(version("typeguard").split(".")[:3])
                                         ^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'split'
```

On CI this fails with a pickling error
```
✨ Pixi task (generate-testmodels in dev): python utils/generate-testmodels.py
Traceback (most recent call last):
  File "/home/runner/work/Ribasim/Ribasim/utils/generate-testmodels.py", line 44, in <module>
    for model_name in p.imap_unordered(generate_model_partial, models):
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/Ribasim/Ribasim/.pixi/envs/dev/lib/python3.12/multiprocessing/pool.py", line [8](https://github.com/Deltares/Ribasim/actions/runs/12147545818/job/33874020840?pr=1959#step:7:9)73, in next
    raise value
multiprocessing.pool.MaybeEncodingError: Error sending result: '<multiprocessing.pool.ExceptionWithTraceback object at 0x7ff[10](https://github.com/Deltares/Ribasim/actions/runs/12147545818/job/33874020840?pr=1959#step:7:11)76f0f80>'. Reason: 'PicklingError("Can't pickle <class 'ribasim.input_base.TableModel[BasinProfileSchema]'>: attribute lookup TableModel[BasinProfileSchema] on ribasim.input_base failed")'
```

I tried rolling back from pandera 0.21 to 0.20 but it didn't seem to help.

**Comment by evetion** [comment]
I can reproduce the CI error, not your typeguard one.

**Comment by visr** [comment]
Yeah neither can I, that is just pixi environment installation sometimes going wrong leading to this kind of error.

File: pixi.toml
Comment (evetion) [inline pixi.toml:1]: You're getting old and conservative mate. 🤗

Comment (visr) [reply pixi.toml:1]: 😭

**Comment by evetion** [comment]
:tada

**Review by evetion — APPROVED** [review]
Can you make an issue (if not already done so) to remove these conda workarounds in all three places?

**Pull request merged**